src/pages/home/report/ReportActionsList.js

```diff
@@ -134,8 +134,8 @@ class ReportActionsList extends React.Component {
         item,
         index,
     }) {
-        const shouldDisplayNewIndicator = this.props.report.newMarkerSequenceNumber > 0
-            && item.action.sequenceNumber === this.props.report.newMarkerSequenceNumber;
+        const shouldDisplayNewIndicator = this.props.newMarkerSequenceNumber > 0
```

Comment:
Is the first condition needed here? can both `sequenceNumber` and `newMarkerSequenceNumber` be equal to zero simultaneously?

Reply:
It looks like we explicitly set this to zero, could we set to null to indicate the lack of a value here instead? NAB but since zero could be a valid sequence number (just based on the data type), null could be a little cleaner

Reply:
Not sure if I understand the question. But the newMarkerSequenceNumber can be equal to zero and the created action has a sequenceNumber of 0. So we do this to avoid showing the marker for the "created" action. We don't want to show the marker when the last read sequenceNumber is 0.

As for using `null` instead, I don't have strong feelings on it, but maybe we can add a comment here at least to explain what I just mentioned.

Reply:
Let me know if this comment helps. Or if you feel strongly that we should use `null`. I'm inclined to leave it since everything is working well for me.